backup/statistics.go

```diff
@@ -230,11 +230,11 @@ func generateAttributeSlotsQuery6(attStat AttributeStatistic) string {
 			realValues(attStat.Numbers3),
 			realValues(attStat.Numbers4),
 			realValues(attStat.Numbers5),
-			AnyValues(attStat.Values1, attStat.Type),
-			AnyValues(attStat.Values2, attStat.Type),
-			AnyValues(attStat.Values3, attStat.Type),
-			AnyValues(attStat.Values4, attStat.Type),
-			AnyValues(attStat.Values5, attStat.Type))
+			AnyValues(attStat.Values1, attStat.Type, attStat.Kind1),
+			AnyValues(attStat.Values2, attStat.Type, attStat.Kind2),
+			AnyValues(attStat.Values3, attStat.Type, attStat.Kind3),
+			AnyValues(attStat.Values4, attStat.Type, attStat.Kind4),
```

Comment:
As I understand, other statvalues arrays depends on the column type of the table. I'm right?
But the `statvalues4`, and `statvalues5` arrays always have the `bytea` type, does we need to pass kind to the AnyValues function, I mean that if we know that type of these arrays doesn't depend on the column type, maybe pass `bytea` type instead of passing kind?

Reply:
> statvalues arrays depends on the column type of the table. I'm right?

May be

Reply:
> the `statvalues4`, and `statvalues5` arrays always have the `bytea` type,

Where does this conclusion come from? I only found that `stavalues5` can contain `bytea[]` if `stakind5` is 99 or 98.

Reply:
Hm... It is seems that [you are right](https://github.com/arenadata/gpdb/blob/030faa2e79cbeb42ab4c843f8c8475c322506990/src/backend/commands/analyze.c#L1284-L1305).

Reply:
fixed